links/link.go

```diff
@@ -426,6 +426,13 @@ func SetNameMACAndUpInterface(l netlink.Link, endpt Endpoint) func(ns.NetNS) err
 			}
 		}
 
+		if endpt.GetIfaceAlias() != "" {
+			err := netlink.LinkSetAlias(l, endpt.GetIfaceAlias())
+			if err != nil {
+				return err
+			}
+		}
+
```

Comment:
@vista- 
instead of link aliases we should use link AltNames

We switched from using Aliases to using AltNames (https://github.com/srl-labs/containerlab/pull/1908) since this makes it easier to query links by name with a single netlink call, while altnames provide all the benefits we are looking for (long names)

https://x.com/ntdvps/status/1763493657804427299

Reply:
mmm. I see why you did that

```
and the forbidden character '/', which many NOS' interface naming conventions favor (and altnames still don't allow us to use them)
```

I didn't know about altnames not allowing `/`